website/docs/guides/best-practices/environment-setup/1-env-guide-overview.md

```diff
@@ -39,7 +39,7 @@ Environments define the way that dbt will execute your code, including:
   - **Development** — the environment settings in which you work in the IDE on a development branch.
   - **Deployment** — the environment settings in which a dbt Cloud job runs.
 
-In this guide, we’re going to focus on **deployment environments**, which determine how your project is executed when a **dbt Cloud job executes**.
+In this guide, we’re going to focus on **deployment environments**, which determine how your project is executed when a **dbt Cloud job executes**. For both approaches, be sure to set one environment as production in order to use features like dbt Explorer and cross-project refs.  
```

Comment:
```suggestion
In this guide, we’re going to focus on **deployment environments**, which determine how your project is executed when a **dbt Cloud job executes**. When using both approaches, make sure to designate one environment as "Production." This will allow you to use features such as dbt Explorer and cross-project references.
```